From 97711723fb3d695c87133b2ec6ab5a9b6dea352c Mon Sep 17 00:00:00 2001
From: Stijn Segers <foss@volatilesystems.org>
Date: Fri, 23 Sep 2022 23:54:15 +0200
Subject: [PATCH 11/18] mvebu: cortexa72: performance as default scheduler

It looks more and more like demand-based governors cause extra
jitter, so ditch 'schedutil' and switch to 'performance'

Signed-off-by: Stijn Segers <foss@volatilesystems.org>
---
 target/linux/mvebu/cortexa72/config-5.15 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/target/linux/mvebu/cortexa72/config-5.15 b/target/linux/mvebu/cortexa72/config-5.15
index 0f19de730e..5d7b5594aa 100644
--- a/target/linux/mvebu/cortexa72/config-5.15
+++ b/target/linux/mvebu/cortexa72/config-5.15
@@ -38,6 +38,9 @@ CONFIG_ARM_PSCI_FW=y
 CONFIG_ARM_SBSA_WATCHDOG=y
 CONFIG_AUDIT_ARCH_COMPAT_GENERIC=y
 CONFIG_CC_HAVE_STACKPROTECTOR_SYSREG=y
+# CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_LITTLE_ENDIAN=y
 CONFIG_DMA_DIRECT_REMAP=y
 CONFIG_FRAME_POINTER=y
-- 
2.42.1

